src/test/java/com/tvtracker/enterprise/EnterpriseApplicationTests.java

```diff
@@ -20,28 +20,29 @@
 
     final String TEST_USERNAME = "testUser";
     final String TEST_USER_PASSWORD = "testPassword";
+    MediaEntry mediaEntry = new MediaEntry();
+    UserAccount userAccount = new UserAccount();
 
     @Test
     void contextLoads() {
     }
 
     @Test
     void userCreatesUserAccount_ReturnsValidAuthenticationToken() throws Exception {
-        UserAccount userAccount = whenUserSendsUserAccountWithUniqueUsername();
+        userAccount = whenUserSendsUserAccountWithUniqueUsername();
         returnsUserAccountWithValidToken(userAccount);
     }
 
     private UserAccount whenUserSendsUserAccountWithUniqueUsername() throws Exception {
-        UserAccount userAccount = new UserAccount();
         userAccount.setUsername(TEST_USERNAME);
         userAccount.setPassword(TEST_USER_PASSWORD);
         userAccount.setEmail("testUser@testSite.com");
 
-        UserAccount newUserAccount = userAccountService.createUserAccount(userAccount);
+        userAccount = userAccountService.createUserAccount(userAccount);
```

Comment:
No need to create a new object as you can just use the setter methods to update the properties of an already existing object. This will eliminate clutter which in term lessens technical debt.
https://martinfowler.com/bliki/TechnicalDebt.html